extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -220,6 +228,19 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.analyticsGroup_ =
         this.instrumentation_.createAnalyticsGroup(this.element);
 
+    if (this.config_['transport'] && this.config_['transport']['iframe']) {
```

Comment:
Done when putting stuff into this.config_ rather than when reading from it. See lines 424-429. (I know you've already found that since you commented on it, just keeping the bookkeeping up to date.)